charts/seed-bootstrap/charts/loki/values.yaml

```diff
@@ -67,7 +67,7 @@ hvpa:
     cpu: 800m
     memory: 2G
   minAllowed:
-    cpu: 20m
+    cpu: 200m
```

Comment:
initial `request` is changed to `100m`. `minAllowed` is changed to `200m`, and `minChange` is min(`100m`, `80% of requests = 80m`).
VPA is immediately going to recommend `200m` because of `minAllowed`, and because it is matching `minChange`, loki is going to be immediately scaled to `200m`.
we should either increase `minChange` or set initial `request` to `200m`.